packages/fxa-auth-server/lib/routes/utils/docs/account_docs.ts

```diff
@@ -0,0 +1,267 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import dedent from 'dedent';
+
+const ACCOUNT_CREATE_POST = {
+  notes: [
+    dedent(`
+      Creates a user account. The client provides the email address with which this account will be associated and a stretched password. Stretching is detailed on the [**onepw**](https://github.com/mozilla/fxa-auth-server/wiki/onepw-protocol#creating-the-account) wiki page.
+
+      This endpoint may send a verification email to the user. Callers may optionally provide the \`service\` parameter to indicate which service they are acting on behalf of. This is an opaque alphanumeric token that will be embedded in the verification link as a query parameter.
+
+      Creating an account also logs in. The response contains a \`sessionToken\` and, optionally, a \`keyFetchToken\` if the url has a query parameter of \`keys=true\`.
+  `),
+  ],
+  plugins: {
+    'hapi-swagger': {
+      responses: {
+        400: {
+          description: dedent`
+            Failing requests may be caused by the following errors (this is not an exhaustive list):
+            \`errno: 101\` - Account already exists
+            \`errno: 144\` - Email already exists
+          `,
+        },
+      },
+    },
+  },
+  tags: ['api', 'Account'],
+};
+
+const ACCOUNT_STUB_POST = {
+  tags: ['api', 'Miscellaneous'],
+};
+
+const ACCOUNT_FINISH_SETUP_POST = {
+  tags: ['api', 'Miscellaneous'],
+};
```

Comment:
You could call this something like `TAGS_MISCELLANEOUS` to DRY it up, and stick it in a shared file. Each type of tag could get its own constant.

You could then have `TAGS_ACCOUNT` similarly, and use that everywhere you need  it, like [L30 ](https://github.com/mozilla/fxa/pull/12266/commits/36397b0315c53298f80f88153705af2dbb611000#diff-de50e4c83d2a5cc5ccdea3f6cf0449c2545c90ee5fe1167bcc8df3482ea8e8c1R30)